cltk/phonology/orthophonology.py

```diff
@@ -104,7 +104,7 @@ class AbstractPhoneme:
 	An abstract phoneme is just a bundle of phonological features.
 	'''
 
-	def __init__(self, features, ipa = None):
+	def __init__(self, features = {}, ipa = None):

```

Comment:
I would be careful with mutable default arguments. Here is a good answer: https://stackoverflow.com/a/41686973.

Reply:
Oh wow.  I didn't realize this!  Thanks for catching it.